modules/meilisearch/meilisearch.go

```diff
@@ -0,0 +1,119 @@
+package meilisearch
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net"
+	"time"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	defaultMasterKey = "just-a-master-key-for-test"
+	defaultHTTPPort  = "7700/tcp"
+	masterKeyEnvVar  = "MEILI_MASTER_KEY"
+)
+
+// MeilisearchContainer represents the Meilisearch container type used in the module
+type MeilisearchContainer struct {
+	testcontainers.Container
+	MasterKey string
```

Comment:
suggestion: use an internal variable and add a method to access, to avoid people thinking assignment of it is valid.